src/utils/current-theme.ts

```diff
@@ -0,0 +1,14 @@
+import { THEME_CONFIG_MAP } from "../themes"
+import { ThemeKey } from "../types/theme"
+
+const THEME_NAMES = Object.keys(THEME_CONFIG_MAP)
+
+export function getCurrentTheme(): ThemeKey {
+  const theme = JSON.parse(localStorage.getItem("theme")!)
```

Comment:
I find it so disconnecting that we set this value to localStorge with `atomWithStorage`

https://github.com/metabase/shoppy/blob/5c4c99b4c650f18bfaebf229004f9e7c15e85dc3/src/store/theme.ts#L6

But we read it from localStorage directly here.

I'd kind of expect you'd try to read it from the atom somehow here, otherwise, this looks like a hack to me.

I found we could use `store` to [interact with atom outside React](https://jotai.org/docs/guides/using-store-outside-react). But I'm not totally sure how to apply that to this scenario, since I'm not familiar with Jotai.

Reply:
Great catch! Replaced this with `store.get` 🙏🏻